resources/Materials/TestSuite/stdlib/udim/multi_udim_bunny.mtlx

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+<materialx version="1.38">
+
+  <image name="image_color" type="color3">
+    <input name="file" type="filename" value="resources/Images/udim_bunny/grid_<UDIM>.png" colorspace="srgb_texture" />
+  </image>

```

Comment:
Very good work. This provides a working test platform for UDIMs.

Try `tiledimage` or `UsdUVTexture` instead of `image` here to observe issue #1716 potentially fixed by #1864 for GLSL if that branch has not become too stale.